src/mixxx.cpp

```diff
@@ -598,6 +596,11 @@ void MixxxMainWindow::initializeWindow() {
     Pal.setColor(QPalette::Background, MenuBarBackground);
     m_pMenuBar->setPalette(Pal);
 
+    // restore window size/position
+    QSettings settings("mixxxdj", "mixxx");
```

Comment:
> If you really want me to use cfg file then can you please write the code yourself? I'm not particularly motivated to spend my time on it, as the QSettings approach works well, and is cleaner and more appropriate IMHO.

All of my free time available for Mixxx work is currently allocated to other higher priority projects and bugs unfortunately. 

If this scratches your own itch and it works for you then that's great -- you can maintain your own fork of Mixxx with your patches included. If you'd like it to be merged then it will need to meet our project standards. Once again I appreciate you working on this and wanting to contribute a fix. It would be nice to include this fix in Mixxx if you have the time to improve it.